rmtk/plotting/damage_dist/parse_damage_dist.py

```diff
@@ -59,8 +60,15 @@ def parse_dmg_dist_tax(element):
             damage_states = str(e.text).split()
         elif e.tag == '%sDDNode' % xmlNRML:
             taxonomy, damage_dist = parse_single_damage_dist(e)
-            taxonomies.append(taxonomy)
-            damage_dist_tax[taxonomy] = damage_dist
+            taxo.append(taxonomy)
+            no_tax = len(taxo)
+            for i in range(no_tax):
```

Comment:
If the .split() is used in the **parse_single_damage_dist** function so that it returns each taxonomy string without the newline and whitespace characters, this for loop could be avoided.